spec/brcobranca/boleto/banco_bradesco_spec.rb

```diff
@@ -168,13 +166,13 @@
   end
 
   it 'Gerar boleto nos formatos válidos' do
-    @valid_attributes[:valor] = 2952.95
-    @valid_attributes[:data_documento] = Date.parse('2009-04-30')
-    @valid_attributes[:dias_vencimento] = 0
-    @valid_attributes[:numero_documento] = '75896452'
-    @valid_attributes[:conta_corrente] = '0403005'
-    @valid_attributes[:agencia] = '1172'
-    boleto_novo = described_class.new(@valid_attributes)
+    valid_attributes[:valor] = 2952.95
+    valid_attributes[:data_documento] = Date.parse('2009-04-30')
+    valid_attributes[:dias_vencimento] = 0
+    valid_attributes[:numero_documento] = '75896452'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.